simulation/protocol.go

```diff
@@ -66,18 +66,21 @@ func (e *Simulation) Setup(dir string, hosts []string) (
 // by the server. Here we call the 'Node'-method of the
 // SimulationBFTree structure which will load the roster- and the
 // tree-structure to speed up the first round.
-func (e *Simulation) Node(config *onet.SimulationConfig) error {
-	return e.SimulationBFTree.Node(config)
+func (s *SimulationProtocol) Node(config *onet.SimulationConfig) error {
+	index, _ := config.Roster.Search(config.Conode.ServerIdentity.ID)
```

Comment:
The second argument is not an error, but I can check for a negative index, which indicates an error.